### PR TITLE
Document new configuration options for air-Q integration

### PR DESCRIPTION
**Pull request by Sibgatulin**
## Proposed change

Document new options proposed in a PR for the air-Q integration.
Namely, the following configurations are exposed

1. A choice between fetching the averaged (old behavior) vs. noisy sensor reading from the device
2. A toggle to clip (spuriously) negative sensor values (new and now default functionality)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/109337
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/airq.markdown [hunk 1]
```diff
@@ -21,7 +21,7 @@ The **air-Q** {% term integration %} allows integrating the sensors, provided by
 
 {% include integrations/config_flow.md %}
 
-During the configuration, the user is prompted for the IP address of the {% term device %} or the first 5 characters of the serial number, as well as the device password.
+During the initial configuration, the user is prompted for the IP address of the {% term device %} or the first 5 characters of the serial number, as well as the device password.
```
Comment (c0ffeeca7) [inline source/_integrations/airq.markdown:24]:
```suggestion
```
I realize this sentence was already in this position before this PR. But I think it makes sense to put it before the config flow, as a precondition. This is also done in other integrations where credentials are required.

File: source/_integrations/airq.markdown [hunk 2]
```diff
@@ -86,3 +86,11 @@ PM1, PM25, and PM10 correspond to concentrations of particulates with diameter l
 Virus Index uses CO2 as a proxy for potential aerosol load and can be seen as an indicator of ventilation sufficiency (0 %: insufficient ventilation, 100 %: all fine).
 
 Virtual sensors "Relative Pressure" and "Virus Index" are introduced in firmware v1.80.0 but deactivated by default. You can activate them in the air-Q mobile application under "Advanced settings".
+
+## Additional configuration
+
+After the integration is initialised, the user can configure any of the following two parameter:
```
Comment (c0ffeeca7) [inline source/_integrations/airq.markdown]:
```suggestion
After the integration has been initialized, the user can configure any of the following two parameters:
```

File: source/_integrations/airq.markdown [hunk 3]
```diff
@@ -86,3 +86,11 @@ PM1, PM25, and PM10 correspond to concentrations of particulates with diameter l
...
+
+- **Show values averaged by the device**. Default: on. In its default configuration, air-Q averages the stream of sensor values, and the strength of this averaging can be configured on the device side (not exposed through the HA). However, this integration allows to switch between polling the averaged and the raw data from the device. To poll noisy sensor readings from the device, switch this feature off.
```
Comment (c0ffeeca7) [inline source/_integrations/airq.markdown]:
```suggestion
- **Show values averaged by the device**. Default: `on`. In its default configuration, air-Q averages the stream of sensor values. The strength of this averaging can be configured on the device side (not exposed through the HA). However, this integration allows to switch between polling the averaged and the raw data from the device. To poll noisy sensor readings from the device, set **Show values averaged by the device** to `off`.
```

File: source/_integrations/airq.markdown [hunk 4]
```diff
@@ -86,3 +86,11 @@ PM1, PM25, and PM10 correspond to concentrations of particulates with diameter l
...
+
+- **Clip negative values**. Default: on. For baseline calibration purposes, certain sensor values may briefly become negative. The default behavior is to clip such values to 0.
```
Comment (c0ffeeca7) [inline source/_integrations/airq.markdown]:
```suggestion
- **Clip negative values**. Default: `on`. For baseline calibration purposes, certain sensor values may briefly become negative. The default behavior is to clip such values to 0.
```

File: source/_integrations/airq.markdown [hunk 5]
```diff
@@ -21,7 +21,7 @@ The **air-Q** {% term integration %} allows integrating the sensors, provided by
 
 {% include integrations/config_flow.md %}
```
Comment (c0ffeeca7) [inline source/_integrations/airq.markdown:23]:
```suggestion

During the initial configuration, the user is prompted for the IP address of the {% term device %} or the first 5 characters of the serial number, as well as the device password.

```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Some tweaks

**Comment by Sibgatulin** [comment]
All suggestions make sense to me, thanks @c0ffeeca7!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Sibgatulin :+1:

**Pull request merged**